langchain/src/tools/tests/outlookIntegration.test.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  AuthFlowToken,
+  OutlookReadMailTool,
+  OutlookSendMailTool,
+} from "../outlook/index.js";
+
+describe("OutlookReadMailTool Test", () => {
+  test("Test invalid access token", async () => {
+    const accessToken = "blah";
+    const authFlow = new AuthFlowToken(accessToken);
+    const outlookTool = new OutlookReadMailTool(authFlow);
+    const emails = await outlookTool._call("");
+    expect(emails).toBe("Fetch mail error: 401");
+  });
+
+  test("Test read messages", async () => {
+    const outlookTool = new OutlookReadMailTool(undefined, "refresh");
+    const emails = await outlookTool._call("");
+    console.log(emails);
+    expect(true).toBe(true);
```

Comment:
Please change these assertions, to check `emails`

Reply:
Will change to `expect(emails.substring(0, 7)).toBe("subject");` to show that email is successfully read, otherwise the first 6 characters wouldn't be subject.